class-tumblr-import.php

```diff
@@ -639,16 +774,61 @@ public function do_pages_import( $url ) {
 		}
 
 		/**
-		 * Handles the sideload import.
+		 * Handles importing any Media within the imported Post
+		 *
+		 * @param  array $post the (already) imported WP_Post
+		 *
+		 * @return void
```

Comment:
will `$this->{$sideload_method}( $post );`  never return? if those methods don't return anything you could just call it without `return`

Reply:
Good catch. None of these methods return anything, they just make updates.